zc_common/jwt_auth/authentication.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, **kwargs):
             setattr(self, kwarg, kwargs[kwarg])
 
     def is_authenticated(self):
-        return 'user' in self.roles or 'service' in self.roles
+        return True
```

Comment:
> The return value of is_authenticated() does not affect whether a request is authenticated by this class or not.

What do you mean? In my understanding, I would say `authentication` is the process of parsing & validating a JWT token to create a User instance. Checking whether the created user is `anonymous` or not falls within the scope of permissions.